samples/supervision/build.gradle

```diff
@@ -41,9 +38,13 @@ android {
   }
 
   compileOptions {
-    kotlinOptions.freeCompilerArgs += [
-        "-Xjvm-default=all",
-    ]
+    sourceCompatibility JavaConfig.code
+    targetCompatibility JavaConfig.code
+  }
+
+  kotlinOptions {
+    jvmTarget = JavaConfig.name
+    freeCompilerArgs += "-Xjvm-default=all"
```

Comment:
I'd prefer if this was appended using the list and previous formatting to avoid git change highlights if additional arguments are applied.

Reply:
fixed in all places in last commit